src/views/portfolio/projects/ProjectCreateProjectModal.vue

```diff
@@ -186,10 +199,31 @@
           this.$toastr.w(this.$t('condition.unsuccessful_action'));
         });
       },
+      retrieveParents: function() {
+        let url = `${this.$api.BASE_URL}/${this.$api.URL_PROJECT}`;
+        this.axios.get(url).then((response) => {
+          for (let i = 0; i < response.data.length; i++) {
+            let project = response.data[i];
+            if (project.version) {
+              this.availableParents.push({value: project.uuid, text: project.name + ' : ' + project.version});
+            } else {
+              this.availableParents.push({value: project.uuid, text: project.name});
+            }
+            if (this.project.parent && this.project.parent.uuid === project.uuid ) {
+              this.selectedParent = project.uuid;
+            }
+          }
+        }).catch((error) => {
+          this.$toastr.w(this.$t('condition.unsuccessful_action'));
+        });
+      },
       resetValues: function () {
         this.project = {};
         this.tag = "";
         this.tags = [];
+        this.selectedParent = null;
+        this.availableParents = [{ value: null, text: ''}]
+        this.retrieveParents();
```

Comment:
It is necessary to reset the available parents, because if a new project is created, it won't show up in the list of selectable parents unless the page is refreshed. But only resetting the available parents after the successful creation of a new project would be better practice.

Reply:
@rbt-mm Even better would be to only load the list when its needed, means, when the dialog is being opened to create/update a project. Before it is not necessary but will always be loaded anyways, correct?

Reply:
Yes, it will always be loaded on the startup of the project list. This procedure was copied from the already existing `retrievelicenses` method, which are also retrieved via an API call on the startup of the project list.
The method can be changed to only occur on opening the dialog, if wanted.